### DISCUSSION TITLE
reactive unknown markdown.

### DISCUSSION THREAD

**Discussion by aduarterengifo** [pngwn/MDsveX#394] [Q&A]
Hello, I was wondering if it's possible to make the following work.

```
<script>
    const hello = "# hello";
</script>

{hello}
```

This works

```
<script>
    const hello = "hello";
</script>

# {hello}
```

I'm trying to preprocess arbitrary markdown from an api.

edit: nvm i figured it out.

```
<script>
    const hello = "# hello";
</script>

{@html hello}
```

**Comment by aduarterengifo** [comment]
nvmx2 it doesn't actually work.


this works:
```
<script>
    const hello = "<h1>hello</h1>";
</script>

{@html hello}
```

this doesn't:
```
<script>
    const hello = "# hello";
</script>

{@html hello}
```

**Comment by pngwn** [comment]
No, this will never work. This is all happening an runtime while mdsvex operates at build time. Some people have some similar thing to this though, I'll turn this into a discussion as it may receive more helpful responses.
